scripts/parse.js

```diff
@@ -37,7 +37,10 @@ const parseModules = dir => {
       } else if (accepted(entryPath)) {
         let filePath = path.join(...entryPath.split("/").splice(2));
         let content = fs.readFileSync(entryPath, "utf8");
-        callback(filePath, content, meta(entryPath));
+        if (filePath === "preview.png") {
+          content = "data:image/png;base64," + fs.readFileSync(entryPath, "base64");
+        }       
+        callback(filePath, content, meta(entryPath));      
```

Comment:
```suggestion
        callback(filePath, content, meta(entryPath));
      } else if (preview(entryPath)) {
        let content = `data:image/png;base64,${fs.readFileSync(entryPath, "base64")}`;
        callback(filePath, content, false, true);
      }
```

Reply:
This new `else if` would validate the path against the `PREVIEW_IMAGE` constant. And it would call the updated `parseModule` callback with the new argument `image = false`.